test/module/irohad/consensus/yac/yac_test_util.hpp

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_YAC_TEST_UTIL_HPP
+#define IROHA_YAC_TEST_UTIL_HPP
+
+#include <gmock/gmock.h>
+
+#include "consensus/yac/vote_message.hpp"
+#include "consensus/yac/yac_hash_provider.hpp"
+
+#include "module/irohad/consensus/yac/mock_yac_crypto_provider.hpp"
+#include "module/shared_model/interface_mocks.hpp"
+
+namespace iroha {
+  namespace consensus {
+    namespace yac {
+
+      inline std::shared_ptr<shared_model::interface::Peer> mk_peer(
+          const std::string &address) {
+        auto key = std::string(32, '0');
+        std::copy(address.begin(), address.end(), key.begin());
+        auto peer = std::make_shared<MockPeer>();
+        EXPECT_CALL(*peer, address())
+            .WillRepeatedly(::testing::ReturnRefOfCopy(address));
+        EXPECT_CALL(*peer, pubkey())
+            .WillRepeatedly(::testing::ReturnRefOfCopy(
+                shared_model::interface::types::PubkeyType(key)));
+
+        return peer;
+      }
+
+      inline VoteMessage createVote(YacHash hash, const std::string &pub_key) {
```

Comment:
This method is the same as `MockYacCryptoProvider::getVote(..)` except for the `pub_key` parameter. Is it possible to join them into one, making the `pub_key` a default parameter with empty string value?

Reply:
I don't see a reason to do that. It is okay that one function is expressed by another. Also, functions have different semantic: one for creating signatures and another one for votes.